specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -9154,6 +9154,16 @@
         "pauseTimeBetweenBatches": {
           "type": "string",
           "description": "The wait time between completing the update for all virtual machines in one batch and starting the next batch. The time duration should be specified in ISO 8601 format. The default value is 0 seconds (PT0S)."
+        },
+
+        "enableCrossZoneUpgrade": {
+          "type": "boolean",
+          "description": "Enable rolling upgrade across multiple zones during UD walk within the limit of MaxBatchInstanceCount or BatchInstancePercent"
+        },
+
+        "prioritizeUnhealthyInstances": {
+          "type": "boolean",
+          "description": "Enable rolling upgrade to start with the batch which has most unhealthy VMs in percentage. If enableCrossZoneUpgrade is not enabled, the rolling upgrade will start from the most unhealthy availability zone,and within the zone, walk the UD that has the most unhealhty VMs first"
```

Comment:
"description": Upgrade all unhealthy instances in a scale set before any healthy instances

Reply:
done